Source/WebKit/UIProcess/LegacyGlobalSettings.h

```diff
@@ -50,8 +50,10 @@ class LegacyGlobalSettings {
     const HashSet<String>& schemesToRegisterAsLocal() { return m_schemesToRegisterAsLocal; }
     void registerURLSchemeAsLocal(const String& scheme) { m_schemesToRegisterAsLocal.add(scheme); }
 
+#if !PLATFORM(COCOA)
     const HashSet<String>& schemesToRegisterAsNoAccess() { return m_schemesToRegisterAsNoAccess; }
     void registerURLSchemeAsNoAccess(const String& scheme) { m_schemesToRegisterAsNoAccess.add(scheme); }
+#endif
```

Comment:
Seems a bit excessive to compile all of this out if we just want to avoid sending it over IPC, but I guess that's ok.